guidelines/index.html

```diff
@@ -1853,24 +1853,24 @@ <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.2</h1
 </section>
 
             
-            <section class="guideline" id="redundant-entry"><div class="header-wrapper"><h4 id="x3-3-7-redundant-entry"><bdi class="secno">Success Criterion 3.3.7 </bdi>Redundant Entry</h4><a class="self-link" href="#redundant-entry" aria-label="Permalink for Section 3.3.7"></a></div>
+            <section class="guideline" id="redundant-entry"><div class="header-wrapper"><h4 id="x3-3-7-redundant-entry"><bdi class="secno">達成基準 3.3.7 </bdi>冗長な入力</h4><a class="self-link" href="#redundant-entry" aria-label="Permalink for Section 3.3.7"></a></div>
    
     
     
     <div class="doclinks"><a href="https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html">Understanding Redundant Entry</a> <span class="screenreader">|</span> <br><a href="https://www.w3.org/WAI/WCAG22/quickref/#redundant-entry">How to Meet Redundant Entry</a></div><p class="conformance-level">(レベル A)</p>
 	<p class="change">[New]</p>
     
-    <p>Information previously entered by or provided to the user that is required to be entered again in the same <a href="#dfn-processes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-processes-2" title="ある活動を完了させるために必要な利用者の一連の動作。">process</a> is either:</p>
+    <p>同一の<a href="#dfn-processes" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-processes-2" title="ある活動を完了させるために必要な利用者の一連の動作。">プロセス</a>において再入力する必要がある、利用者によって以前に入力された、又は利用者に提供された情報は、以下のいずれかである。</p>
 
     <ul>
-        <li>auto-populated, or</li>
-        <li>available for the user to select.</li>
+        <li>自動入力される。又は、</li>
+        <li>利用者が選択可能である。</li>
     </ul>
-    <p>Except when:</p> 
+    <p>ただし、以下の場合は例外とする。</p> 
     <ul>
-        <li>re-entering the information is <a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-16" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">essential</a>,</li>
-        <li>the information is required to ensure the security of the content, or</li>
-        <li>previously entered information is no longer valid.</li>
+        <li>情報の再入力が<a href="#dfn-essential" class="internalDFN" data-link-type="dfn" id="ref-for-dfn-essential-16" title="もし取り除いてしまうと、コンテンツの情報又は機能を根本的に変えてしまい、かつ、適合する他の方法では情報及び機能を実現できない。">必要不可欠</a>である。</li>
+        <li>その情報がコンテンツの安全性を確保するために必要である。又は、</li>
```

Comment:
ここは素直にセキュリティでよいのではないかなと。

```suggestion
        <li>その情報がコンテンツのセキュリティを確保するために必要である。又は、</li>
```